app/controllers/hub/tax_returns_controller.rb

```diff
@@ -14,8 +14,7 @@ def edit; end
     def show; end
 
     def update
-      @tax_return.update!(assign_params)
-      SystemNote.create_assignment_change_note(current_user, @tax_return)
+      @tax_return.assign!(assigned_user_id: assign_params[:assigned_user_id], assigned_by: current_user)
```

Comment:
I find this confusing, but I see why it's there. Maybe it's peaceful to do `.merge(assigned_by: current_user)` at the end of `def assign_params`?

Not a blocker.

Reply:
I started with that approach, but it ended up looking a lot uglier and trickier looking in my opinion. I think its weird to pass raw params or a permitted params object into an instance method in the controller, so i was having to do **assign_params.to_h.symbolize_keys! in order to get it passed into the method correctly.